nest-tax-backend/src/card-payment-reporting/card-payment-reporting.service.ts

```diff
@@ -327,10 +328,17 @@ export class CardPaymentReportingService {
         if (!fileDate) return null
 
         const dateInfo = this.getDateInfo(fileDate)
+        if (dateInfo.today.isBefore(dayjs().subtract(180, 'day'))) {
+          return null
+        }
 
         // Parse CSV and add columns
         const csvData = this.processCsvData(file.content)
 
+        if (csvData.length >= 2 && csvData[1].transactionType === 'POHL') {
+          return null
+        }
```

Comment:
From data we have, it seems to be only case if there are exactly 2 entries. I would much rather for this to fail, then to be ignored if something changes and there will be extra lines after the second line. Therefore strict check for exactly 2 entries is needed, `csvData.length == 2`.